utils/kind/cluster.yaml

```diff
@@ -0,0 +1,9 @@
+---
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    image: kindest/node:v1.30.0
+    extraMounts:
+      - hostPath: $(WASM_PATH)
```

Comment:
this file is a template, which is rendered with the right wasm path. This adds some (minor) extra complexity. 

Have you considered volume source being a secret which could be easier to manage and monitor?

Reply:
My idea was to reduce the number of changes required to iterate as developing - so a simple rebuild of the `make build` has generated a new file in the kind cluster.

That being said it's still pretty simple to re-create the secret each time we rebuild so I'm not tied to one approach or the other. If you prefer the secret I can look into it